packages/firebase_admob/android/src/main/java/io/flutter/plugins/firebaseadmob/MobileAd.java

```diff
@@ -150,8 +154,15 @@ void show() {
         LinearLayout content = new LinearLayout(activity);
         content.setId(id);
         content.setOrientation(LinearLayout.VERTICAL);
-        content.setGravity(Gravity.BOTTOM);
+        content.setGravity(anchorType == 0 ? Gravity.BOTTOM : Gravity.TOP);
         content.addView(adView);
+        final float scale = activity.getResources().getDisplayMetrics().density;
```

Comment:
Can the scaling be done on the Dart side (to avoid doing it in both Java and ObjC)?